### PR TITLE
chore: drop xunit remainder

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: src/Playwright.Tests/Playwright.Tests.csproj [hunk 1]
```diff
@@ -7,21 +7,23 @@
         <NoWarn>1701;1702</NoWarn>
         <AssemblyName>Microsoft.Playwright.Tests</AssemblyName>
         <RootNamespace>Microsoft.Playwright.Tests</RootNamespace>
+        <GenerateProgramFile>false</GenerateProgramFile>
     </PropertyGroup>
     <Import Project="../Common/SignAssembly.props" />
     <ItemGroup>
+		<PackageReference Include="CommandLineParser" Version="2.9.0-preview1" />
```
Comment (avodovnik) [inline src/Playwright.Tests/Playwright.Tests.csproj:14]: Formatting.

**Review by avodovnik — CHANGES_REQUESTED** [review]
The `PlaywrightTestAttribute` needs to be accessible from the tooling, otherwise it doesn't work. I've got a PR that I'm already working on, that will fix this correctly. I'd almost suggest you close this one, and I'll integrate the rest of your suggested changes (the pupeteer mention removals).

File: src/Playwright.Tests/Playwright.Tests.csproj (hunk 1 above)
Comment (pavelfeldman) [reply src/Playwright.Tests/Playwright.Tests.csproj]: I copied it from one file to another, looks like our formatting is inconsistent.

**Comment by pavelfeldman** [comment]
> The PlaywrightTestAttribute needs to be accessible from the tooling, otherwise it doesn't work. 

I don't think you followed my change!

File: src/Playwright.Tests/Playwright.Tests.csproj (hunk 1 above)
Comment (avodovnik) [reply src/Playwright.Tests/Playwright.Tests.csproj]: Yeah, I think that's VS Mac vs VS inconsistencies.

**Review by avodovnik — APPROVED** [review]

**Pull request merged**